internal/crypto/key_derivation_testing.go

```diff
@@ -10,24 +10,25 @@ import (
 )
 
 const (
-	// DefaultKeyDerivationAlgorithm is the key derivation algorithm for new configurations.
-	DefaultKeyDerivationAlgorithm = "testing-only-insecure"
-
 	// MasterKeyLength describes the length of the master key.
 	MasterKeyLength = 32
 
-	V1SaltLength    = 32
-	HashVersion1    = 1 // this translates to Scrypt KeyDerivationAlgorithm
+	V1SaltLength = 32
+
 	ScryptAlgorithm = "scrypt-65536-8-1"
-	Pbkdf2Algorithm = "pbkdf2"
+
+	Pbkdf2Algorithm = "pbkdf2-sha256-600000"
+
+	// DefaultKeyDerivationAlgorithm is the key derivation algorithm for new configurations.
+	DefaultKeyDerivationAlgorithm = ScryptAlgorithm
```

Comment:
nit: it seems that the default should be "testing-only-insecure"

Reply:
This breaks tests, since we have separated the hash versions from the key derivation algorithms.
I would have to handle `testing-only-insecure` in the `internal/user` package but that would mean introducing testing code in production

Reply:
OK